docs/docs/analytics.mdx

```diff
@@ -0,0 +1,126 @@
+---
+id: analytics
+sidebar_label: Analytics
+title: Analytics
+description:
+abstract: Rasa Pro Analytics is an ETL pipeline that extracts model, training, and conversation data from the Rasa platform and creates a queryable and extensible database schema.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+:::warning
+TODO: Lots of content missing, more a structure atm.
+:::
+
+You can query and
+visualise metrics in a frontend tool of your choice (e.g. Metabase, PowerBI)
+
+Measure the success of a Rasa assistant and making strategic investments
+will lead to better business outcomes. It will also help you understand
+the needs of your end users and better serve them.
+
+Rasa Pro Analytics will start analyzing your conversations as soon as
+the docker container is successfully deployed
+and connected to your database and Kafka instances.
+
+## Getting Started
+
+### Ingesting past Conversational data
+
+- if you've previously already streamed the data of your assistant to Kafka
+  and Kafka is configured to retain events on this topic, you can
+  connect Analytics to your Kafka instance and it will process all prior
+  events.
+- if not, ingest the data using
+
+  ```shell
+  rasa export --endpoints endpoints.yml
+  ```
+
+  The export needs to be configured to read from your production tracker store
+  and write to Kafka as an event broker, e.g.
+
+  ```yaml-rasa title="endpoints.yml"
+    tracker_store:
+        type: SQL
+        dialect: "postgresql"
+        url: "localhost"
+        db: "tracker"
+        username: postgres
+        password: password
+
+    event_broker:
+        type: kafka
+        topic: rasa-events
+        url: localhost:29092
+        partition_by_sender: true # VERY IMPORTANT!!!!
+
+  ```
+
+### Connecting your Business Intelligence Solution
+
+#### Example: Connecting Metabase
+
+[Connecting Metabase to Redshift](https://www.metabase.com/data_sources/amazon-redshift)
+
+#### Example: Connecting Tableau to Rasa Pro Analytics
+
+[Connecting Tableau to Redshift](https://help.tableau.com/current/pro/desktop/en-us/examples_amazonredshift.htm)
+
+## Example Queries
```

Comment:
We'll probably need a comprehensive section on all the Metrics that can be queried (without necessarily writing the SQL code for each one?)

Reply:
I am not sure I understand what you mean - how is that different from the section documentation the tables and fields?

Reply:
I actually had in mind this [list of metrics](https://www.notion.so/rasa/Analytics-terms-fce3aa5f8d9341a8a5c9153f0cb9086e)

Reply:
I think for the ones where we can write the SQL we should and for the others we can at least provide a pointer on how to approach them, maybe in a table